lib/spack/spack/traverse.py

```diff
@@ -320,6 +376,11 @@ def traverse_edges(
         A generator that yields ``DependencySpec`` if depth is ``False``
         or a tuple of ``(depth, DependencySpec)`` if depth is ``True``.
     """
+
+    # TODO: cover=edges for order=topo? does visited make sense?
```

Comment:
For now, I think it makes sense to prohibit `cover="edges"` for `order="topo"`: I don't know of an interpretation of topological ordering which applies for edges.

Reply:
In the latest commits `cover=edges` + `order=topo` generates all edges of the DAG ordered s.t. all in-edges per vertex precede its out-edges.

Reply:
I'm generally reluctant to implement something we have no known use case for, but does not implementing this cause some other issue (i.e. a failure since the code using this makes assumptions about what is available in the interface)?